schema/deploy/migrations/insert_json_schema_form_data.sql

```diff
@@ -1,34 +1,40 @@
+-- Deploy cif:migrations/insert_json_schema_form_data to pg
```

Comment:
Should we put all data files in a `schema/deploy/data/<...>_data.sql` ? to separate it from migrations on existing user data in the database

Reply:
I _think_ putting something under deploy that isn't in the sqitch plan will make sqitch angry?

Reply:
I didn't express myself well, I just meant updating the path for this file to be in `schema/deploy/data/...` instead of `schema/deploy/migrations/...`